tests/default/list/indices.yaml

```diff
@@ -0,0 +1,151 @@
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    response:
+      status: 200
+      content_type: text/plain
+  - synopsis: List information related to one index (text/plain).
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices/{index}
+    parameters:
+      index: books
+    response:
+      status: 200
+      content_type: text/plain
+  - synopsis: List information related to indexes with verbose output (v=true).
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      v: true
+    response:
+      status: 200
+      content_type: text/plain
+  - synopsis: List information related to indexes with headers (h=header1,header2).
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      h:
+        - health
+        - status
+    response:
+      status: 200
+      content_type: text/plain
+  - synopsis: List information related to indexes displaying all available headers (help=true).
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      help: true
+    response:
+      status: 200
+      content_type: text/plain
+  - synopsis: List information related to indexes with sorted results.
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      s:
+        - status
+    response:
+      status: 200
+      content_type: text/plain
+  - synopsis: List information related to indexes with bytes, time and all fields.
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      format: json
+      bytes: b
+      time: s
+      h:
+        - '*'
+    response:
+      status: 200
+      content_type: application/json
+  - synopsis: List information related to indexes with all fields.
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      format: json
+    response:
+      status: 200
+      content_type: application/json
+  - synopsis: List information related to indexes in different formats (format=json).
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      format: json
+    response:
+      status: 200
+      content_type: application/json
+  - synopsis: List information related to indexes in different formats (format=yaml).
+    version: '>= 2.18'
+    method: GET
+    path: /_list/indices
+    parameters:
+      format: yaml
+    response:
+      status: 200
+      content_type: application/yaml
+  - synopsis: List information related to indexes in different formats (format=cbor).
+    version: '>= 2.18'
+    distributions:
+      excluded:
+        - amazon-managed
+        - amazon-serverless
+    method: GET
+    path: /_list/indices
+    parameters:
+      format: cbor
+    response:
+      status: 200
+      content_type: application/cbor
+  - synopsis: List information related to indexes in different formats (format=smile).
+    version: '>= 2.18'
```

Comment:
Since this whole thing is `>= 2.18`, move that to the top of this file and remove from individual tests.

Reply:
Addressed in the revision.